src/PIL/ImageOps.py

```diff
@@ -77,9 +77,14 @@ def autocontrast(image, cutoff=0, ignore=None, mask=None):
     :param mask: Histogram used in contrast operation is computed using pixels
                  within the mask. If no mask is given the entire image is used
                  for histogram computation.
+    :param preserve_tone: Preserve image tone in Photoshop-like style autocontrast.
```

Comment:
Please can you add a `versionadded` directive to this? The next version will be 8.2.0.

And please add to release notes:

https://github.com/python-pillow/Pillow/blob/master/docs/releasenotes/8.2.0.rst

Reply:
Done